src/db_service/src/db_layer.rs

```diff
@@ -1,9 +1,11 @@
-use anyhow::{self, ensure, format_err};
 use exonum_merkledb::{Database, DbOptions, Fork, RocksDB, Snapshot};
 
 fn create_db_instance() -> RocksDB {
     let db_options: DbOptions = Default::default();
-    RocksDB::open("dbtest/rocksdb", &db_options).unwrap()
+    match RocksDB::open("dbtest/rocksdb", &db_options) {
```

Comment:
Can we read this from config. It had DB Path

Reply:
Sure.